README.md

```diff
@@ -1 +1,71 @@
-# csaf-validator-service
\ No newline at end of file
+<div id="top"></div>
+
+# CSAF Validator Service
+
+<!-- toc -->
```

Comment:
The pre-commit script to update the TOC: Does it also run locally or just on Github?

Reply:
It runs locally only to update the readme toc.